stemcell_builder/stages/image_install_grub/apply.sh

```diff
@@ -78,9 +81,20 @@ then
 GRUB_CMDLINE_LINUX="vconsole.keymap=us net.ifnames=0 crashkernel=auto selinux=0 plymouth.enable=0"
 EOF
 
+  pbkdf2_password=`run_in_chroot ${image_mount_point} "echo -e '${random_password}\n${random_password}' | grub2-mkpasswd-pbkdf2 | grep -Eo 'grub.pbkdf2.sha512.*'"`
```

Comment:
@xingzhou could you please put comments why we configure password for vcap user?